pkg/telemetry/phonehome/telemeter.go

```diff
@@ -8,16 +8,27 @@ type Telemeter interface {
 	// the buffers.
 	Stop()
 	// Identify updates the user traits.
-	Identify(userID string, props map[string]any)
+	Identify(props map[string]any)
 	// Track registers an event, caused by the user.
-	Track(event, userID string, props map[string]any)
+	Track(event string, props map[string]any)
 	// Group adds the user to a group, supplying group specific properties.
-	Group(groupID, userID string, props map[string]any)
+	Group(groupID string, props map[string]any)
+
+	// IdentifyUserAs updates the user traits.
+	IdentifyUserAs(userID, clientID, clientType string, props map[string]any)
+	// TrackUserAs registers an event, caused by the user.
+	TrackUserAs(userID, clientID, clientType, event string, props map[string]any)
+	// GroupUserAs adds the user to a group, supplying group specific properties.
+	GroupUserAs(userID, clientID, clientType, groupID string, props map[string]any)
 }
 
 type nilTelemeter struct{}
 
-func (t *nilTelemeter) Stop()                                              {}
-func (t *nilTelemeter) Identify(userID string, props map[string]any)       {}
-func (t *nilTelemeter) Track(event, userID string, props map[string]any)   {}
-func (t *nilTelemeter) Group(groupID, userID string, props map[string]any) {}
+func (t *nilTelemeter) Stop()                            {}
+func (t *nilTelemeter) Identify(_ map[string]any)        {}
+func (t *nilTelemeter) Track(_ string, _ map[string]any) {}
```

Comment:
Why did you remove the parameter names? I'd leave them for better context.

Reply:
well, they don't matter here, produce noise, and take a lot of space.